docs/usage.md

```diff
@@ -39,6 +39,14 @@ https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/sourmash_
 https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/sourmash_dbs/GCF_003697165.2.db.zip,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/sourmash_dbs/GCF_003697165.2.taxonomy.csv
 ```
 
+Finally, you will need to provide the path to an already prepared DIAMOND database of your choosing. For example, we prepared a DIAMOND database of the Uniprot uniref90 proteins by running:
+
+```
+diamond makedb --in uniref90.fasta.gz --taxonmap prot.accession2taxid.FULL --taxonnodes taxdmp/nodes.dmp --taxonnames taxdmp/names.dmp -d 2023-04-26-uniref90.dmnd
+```
```

Comment:
oh beautiful. Can you link to this in the main page, and include the wget/curl download commands to get uniref90, prot.accessions2taxid.FULL, and the dmp files?

Reply:
addressed